dashboard/src/components/Comments.js

```diff
@@ -223,7 +224,7 @@ const EditingComment = ({ value = {}, commentId, onSubmit, onCancel, onDelete, n
             if (!body.comment) return toast.error('Le commentaire est obligatoire');
             if (body.date && outOfBoundariesDate(body.date)) return toast.error('La date est hors limites (entre 1900 et 2100)');
 
-            await onSubmit({ ...value, ...body });
+            await onSubmit({ ...value, ...body, team: value.team ?? body.team ?? currentTeam._id });
```

Comment:
Sauf erreur on veut préférer `body.team` plutôt que `value.team` (voir l'ordre des arguments `{ ...value, ...body }`) donc ce serait : 

```suggestion
            await onSubmit({ ...value, ...body, team: body.team ?? value.team ?? currentTeam._id });
```